lib/services/intune/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "azure-arm-intune",
+  "author": "Microsoft Corporation",
+  "contributors": [
+    "Watanabe Garcia, Otniel <otnielw@microsoft.com>"
+  ],
+  "version": "0.1.0",
+  "description": "Microsoft Intune Resource Management Client Library for node",
+  "tags": [
+    "azure",
+    "sdk"
+  ],
+  "keywords": [
+    "node",
+    "azure"
+  ],
+  "main": "./lib/intune.js",
+  "engines": {
+    "node": "^5.0.0"
+  },
+  "licenses": [{
+    "type": "Apache 2.0",
+    "url": "http://www.apache.org/licenses/LICENSE-2.0"
+  }],
+  "dependencies": {
+    "ms-rest-azure": "1.2.0",
+    "ms-rest": "1.2.0",
```

Comment:
```"dependencies": {
    "ms-rest-azure": "^1.2.0",
    "ms-rest": "^1.2.0"
  },```

There should not be a comma after the last key-value pair in a json object. Please remove comma after "ms-rest":"1.2.0".
At the same time also add `^` before the version number. Then you need not update the package.json when we publish a newer version of the clientruntime. {`^1.2.0` means greater than or equal to 1.2.0}